docs/Air.md

```diff
@@ -26,6 +26,7 @@ The Air workflow allows you to do what most travel agents did in the past and wh
 * [.getTickets(params)](#getTickets)
 * [.cancelTicket(params)](#cancelTicket)
 * [.cancelBooking(params)](#cancelBooking)
+* [.ackScheduleChange(params)](#ackScheduleChange)
```

Comment:
Please rename to `acknowledge` or `accept` to make it more readable.
I needed to consult vocabulary for abbreviation meaning :)

Reply:
Accept sounds more natural, but the uAPI named ack, so renamed to acknowledgeScheduleChange

Reply:
We don't always follow api naming routine, you know :)
In your another PR you took traveler as it's in uAPI, but we had passengers already, so you may keep accept. I find it more suitable too.

Reply:
So, should we name it to acceptScheduleChange?

Reply:
I would prefer it.

Reply:
renamed it to acceptScheduleChange :)